res/css/_common.pcss

```diff
@@ -51,6 +51,19 @@ limitations under the License.
     --dialog-zIndex-standard: calc(var(--dialog-zIndex-standard-background) + 1); /* 4012 */
 }
 
+/**
+ * Doubling the selector to increase specificity as Compound defines it as such
```

Comment:
I'm not able to make sense of this comment :thinking: Do you mean that we need increased specificity here to override the value that the design tokens package sets?